app/views/layouts/_google_analytics.html.erb

```diff
@@ -1,12 +1,11 @@
 <% if ENV['GOOGLE_ANALYTICS_ID'].present? %>
+<!-- Global Site Tag (gtag.js) - Google Analytics -->
+<script async src='<%= "https://www.googletagmanager.com/gtag/js?id=#{ENV['GOOGLE_ANALYTICS_ID']}" %>'></script>
```

Comment:
Related to @citizen428's point below, it's safer to keep HTML attributes in double quotes. Nothing breaks at the moment, but just in case a single quote sneaks in in the future 😄